Responsiveness Fixed

aliraza944 opened the pull request:
- Resolve #101 

- [x] About page responsiveness resolved
- [x]  Made the iframe to be adaptive on all screens. You can change the height of the iframe to adjust to your suitability.
- [x] Explore data flex layout set the column for small screens and row for large screens.
- [x]  Heights adjusted so that the footer on the about page stays on the bottom

d-rita commented on src/views/Home.js:
```diff
@@ -23,8 +23,8 @@ export function DefaultPage({ children }) {
   return (
-    <div className="h-screen w-100">
-      <div className="h-full w-full absolute z-10 flex items-center">
+    <div className=" w-full">
```
@aliraza944 can we retain the height and width specifications, i.e. "h-screen w-100" in order to be able to display the map? With just "w-full" and no inherited width property to define it, nothing is set and the map does not show.

d-rita reviewed (changes requested):
Tested the PR and the changes are good. The Reports page and the About page are responsive to different screen sizes with no UI elements overlapping each other. 
1. Explore Reports Page:
![explore reports page - tested](https://user-images.githubusercontent.com/31903212/174552509-1eefb346-73e2-4e77-9eca-bfd890eae74c.png)
2. About Page: 
![about page - tested](https://user-images.githubusercontent.com/31903212/174552555-0700decb-ff81-4621-8956-9541405cb0f9.png)

However, the Home page does not have the map displayed with the proposed change in the width and height specs. Reverting to the previous ones will ensure the map is displayed and the page is responsive.

aliraza944 commented:
I have adjusted the height of the Mapbox div. I couldn't get your Mapbox to work but I have fixed the height issue and its responsiveness.<br/ >
I have attached two images with the Mapbox container background to yellow so you can check and verify.
![mobile](https://user-images.githubusercontent.com/50791045/174632232-fa82b34f-56a8-4458-90eb-53c7d194bf8d.png)
![desktop](https://user-images.githubusercontent.com/50791045/174632270-1c4150cb-267c-4f4c-a1fb-267a49beb7ab.png)


**Please let me know if you require anything else**

d-rita commented:
@aliraza944 thank you for your quick response to this. This is the [current production site](https://galaxy.hotosm.org/) and this is the home page design we have. 

Design:
![home page](https://user-images.githubusercontent.com/31903212/174633362-84d39d77-11a4-41b5-b203-b2144dbbdfe9.png)

Retaining the previous width and height specs while maintaining the other proposed changes gave a responsive home page.
![home page tested](https://user-images.githubusercontent.com/31903212/174633863-b455ea50-8d76-4dda-8baa-9506e4625343.png)

aliraza944 commented:
I thought that MapBox div is at the bottom.

d-rita commented:
We have a map background onto which the text and the Explore button appear. Following the[ Mapbox tutorial](https://docs.mapbox.com/help/tutorials/use-mapbox-gl-js-with-react/) here, we render the Mapbox map using the div and add another div with the other components in order to build out the Home page.

aliraza944 commented:
You can test now I have set the banner as absolute and now it appears in front of the map and the map takes all the space.<br />
Here are the images attached for reference.
![desktop](https://user-images.githubusercontent.com/50791045/174646432-803e7784-1884-423c-813b-21f13dbfd0b2.png)
 
![mobile](https://user-images.githubusercontent.com/50791045/174646506-331487ce-9ebe-4c89-bf91-3fb30b99a02f.png)

d-rita reviewed (approved):
Home, About and Explore Report pages are responsive across various screen sizes.

petya-kangalova commented:
@aliraza944  I wanted to say huge thanks for contributing to the Galaxy UI repo. We are very excited to see new contributors to our repo and really appreciate the input!
